src/internal/connector/onedrive/collections.go

```diff
@@ -47,6 +48,12 @@ type folderMatcher interface {
 	Matches(string) bool
 }
 
+const (
```

Comment:
These should be declared in `url_cache.go`

Reply:
Yeah! This is merge aftermath.